README.md

```diff
@@ -88,7 +88,7 @@ Download Shairport Sync:
 Next, `cd` into the shairport-sync directory and execute the following command:
 
 ```
-$ git checkout development
+$ git checkout modify_delay_check
```

Comment:
Thanks Peter! I missed that. I'll fix it now. There also a few typos in the alsa device diagnostics I should fix.